src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -156,7 +156,7 @@ private AIState pickUpResiduals()
     {
         if (getItemsForPickUp() == null)
         {
-            searchForItems();
+            fillItemsList();
```

Comment:
fillItemsList does fill the items list with the result of searchForItems() in the AbstractEntityAIInteract